content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -86,14 +86,19 @@ To account for modern web applications, loading time watches for network request
 
 - **Initial Load**: Loading Time is equal to _whichever is longer_:
 
-  - The difference between `navigationStart` and `loadEventEnd`.
-  - Or the difference between `navigationStart` and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
+  - The difference between `navigationStart` and `loadEventEnd`, or
+  - The difference between `navigationStart` and the first time the page has no activity for 100ms (with activity defined as ongoing network requests or a DOM mutation). Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 - **SPA Route Change**: Loading Time is equal to the difference between the URL change and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 ### How page activity is calculated
 
-The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page activity tracker is calculated through the following process:
+
+- Check ongoing `xhr` or `fetch` requests
+- Check if resources (js, css, and such) ended in a 100ms sliding window
+
+ The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is considered to have activity when:

- `xhr` or `fetch` requests are in progress
- Performance resource timing entries are emitted by the browser (loading end of js, css, and such)
- DOM mutations are emitted by the browser

The page activity is considered ended when the page don't have activity for 100ms.

**Note**: only activity happening after the SDK init is taken into account.
```

Reply:
Thank you, @bcaudan !